src/visualizer.ts

```diff
@@ -2,13 +2,17 @@ import { render, h } from 'preact';
 import { UIRouter, UIRouterPlugin } from '@uirouter/core';
 import { StateSelector } from './selector/StateSelector';
 import { StateVisualizer } from './statevis/StateVisualizer';
-import { StateTree } from './statevis/tree/StateTree';
+import { NodeOptions, StateTree } from './statevis/tree/StateTree';
 import { TransitionVisualizer } from './transition/TransitionVisualizer';
 
 const visualizer = (router: UIRouter) => new Visualizer(router, {});
 
+export interface StateVisualizerOptions {
+  nodeOptions?: NodeOptions;
```

Comment:
Let's name this property `node`.  We already know it's options because it's in the options object.